### PR TITLE
Hybridize dependencies

### PR DESCRIPTION
**Pull request by awikkerink**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -17,10 +17,20 @@
     "web component"
   ],
   "dependencies": {
-    "polymer": "^1.7.0"
+    "polymer": "Polymer/polymer#1.9 - 2"
   },
   "devDependencies": {
-    "d2l-demo-template": "~0.0.5"
+    "d2l-demo-template": "^1.0.1"
```
Comment (dlockhart) [inline bower.json]: I think this should be `^0.0.12 - 1.0.1` and then in the `1.x` variant have `0.0.12`? Similar to:
https://github.com/BrightspaceUI/colors/blob/master/bower.json

File: package.json [hunk 2]
```diff
@@ -30,10 +30,9 @@
   },
   "homepage": "https://github.com/BrightspaceUI/offscreen",
   "devDependencies": {
-    "bower": "^1.7.9",
```
Comment (dlockhart) [inline package.json]: Why no more dependency on bower?

**Comment by dlockhart** [comment]
The `.travis.yml` file should probably also change to `polymer install --variants` instead of `bower install`.

File: package.json (hunk 2 above)
Comment (awikkerink) [inline package.json]: Polymer-cli brings it in, and `polymer install` replaces `bower install`

File: .editorconfig [hunk 3]
```diff
@@ -8,6 +8,6 @@ insert_final_newline = true
 indent_style = tab
 indent_size = 4
 
-[{package.json,.travis.yml}]
+[{bower.json,package.json,.travis.yml}]
```
Comment (dlockhart) [inline .editorconfig]: Maybe `polymer.json` too?

Comment (ryantmer) [inline .editorconfig]: I've been switching a lot to `[{*.json,*.yml}]` which is another option

Comment (dlockhart) [inline .editorconfig]: Yeah that works too!

**Pull request merged**